.rhdh/docs/openshift.adoc

```diff
@@ -26,6 +26,10 @@ image::images/rhdh_from_operator.png[RHDH from Operator]
 
 === Configurations for Operator-backed RHDH
 
+NOTE: At the moment, updates to the Backstage Custom Resource (CR) are automatically handled by the Operator.
+But updates to resources (e.g., ConfigMaps, Secrets) referenced in the CR are currently not handled automatically (unless the CR itself is updated).
+In this case, you will need to manually delete the Backstage Deployment, so that the Operator can recreate it with the updates.
```

Comment:
delete the Deployment including pods, or just delete and recreate the CR? 

what about app config files ref'd by the CR?

Reply:
> delete the Deployment including pods, or just delete and recreate the CR?

Just deleting the Deployment, not the CR. As I've tested, the operator will recreate a Deployment from the CR.

Reply:
> what about app config files ref'd by the CR?

Yes, that was my point with this sentence: "But updates to resources (e.g., ConfigMaps, Secrets) referenced in the CR are currently not handled automatically (unless the CR itself is updated).
In this case, you will need to manually delete the Backstage Deployment, so that the Operator can recreate it with the updates."

So if only the app-config ConfigMap ref'd in the CR is updated, but not the CR itself, the user will have to manually delete the Backstage deployment (until https://github.com/janus-idp/operator/issues/74 is implemented).

Reply:
got it -- wasn't sure if app config was a configmap object. 

You could also phrase this as "scaling down the deployment to 0 and then back to the previous number of pods", but I guess `manually delete the Backstage Deployment` is clear enough.

Reply:
I remember I tried scaling down to 0 then up again, but that would depend on what changes are done in such linked resources. In some cases (e.g., adding a new app-config file as a new key to an existing ConfigMap), that would require updating the Deployment spec (e.g, main container args). So just scaling the existing Deployment wouldn't work.
I found deleting the Deployment so that the operator can recreate a new one fixed that kind of case.